azkaban-common/src/main/java/azkaban/server/HttpRequestUtils.java

```diff
@@ -87,16 +87,17 @@ public static ExecutionOptions parseFlowOptions(final HttpServletRequest req)
     String concurrentOption = "skip";
```

Comment:
Can you just say:
`String concurrentOption = getParam(req, "concurrentOption", "skip");`
Or something like that so we don't need the if statement at all?